src/rounds/game_modes/empty.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Optional
+
+from arena.arena import Arena
+from player import Player
+from rounds.game_modes.game_mode import GameMode
+from sprite_lists import SpriteLists
+
+# from player_manager import PlayerManager
+
+if TYPE_CHECKING:
+    from player import Player
+
+
+class EmptyGameModePlayerState:
+    def __init__(self, player: Player, lives: int):
+        self.lives = lives
+        self.player = player
+
+
+class EmptyGameMode(GameMode):
+    def __init__(self, players: list[Player]):
+        super().__init__()
+        self.players = players
+
+        for player in players:
+            player.game_mode_state = EmptyGameModePlayerState(player, lives=1)
+
+    def on_round_init(self, arena: Arena, sprite_lists: SpriteLists):
+
+        super().on_round_init(arena, sprite_lists)
+
+    def on_round_start(self):
+        pass
+
+    def create_hud(self):
+        pass
+
+    def create_player_hud(self, player: Player):
+        pass
+
+    def update(self, delta_time: float):
+        pass
+
+    def on_player_death(self, player: Player):
+        state = player.game_mode_state
+        state.lives = 1
+        player.allowed_to_respawn = state.lives > 0
+
+    def get_winner(self) -> Optional[Player]:
+
+        return None
```

Comment:
Any method overrides in `EmptyGameMode` that do exactly the same thing as the base class `GameMode` can be removed.  No sense overriding a method if we're not changing its behavior at all.

For example, if we look at `def get_winner` in `GameMode`, we see that it does exactly the same thing as here:
https://github.com/CastIronChat/twisted-metal/blob/9a940158f8a8d02272acfdc516f55b30bf87b759/src/rounds/game_modes/game_mode.py#L62-L68